epictrack-web/src/components/shared/MasterTrackTable/utils.ts

```diff
@@ -9,7 +10,10 @@ export function getSelectFilterOptions<T>(
   // Step 2: Iterate through the data array to populate the Map
   data.forEach((dataObject) => {
     // Step 3: Skip undefined or null values
-    if (dataObject[key] === undefined || dataObject[key] === null) return;
+    if (dataObject[key] === undefined || dataObject[key] === null) {
+      optionsMap.set("", NOT_APPLICABLE);
```

Comment:
are we supposed to have NA on the dropdowns with no proper values? 
In workplan, I can see NA in end_date which has proper values
responsibilities have two NAs

Reply:
Correct if I understand the comment wrong, you mean this right:
![image](https://github.com/bcgov/EPIC.track/assets/91914654/cd35f207-8de7-468a-8344-d47243fcd7e7)
To have NA value when there is no other possible value


In my understanding, NA is a proper value since it can exist in the dropdown. I fear we might over-engineer it if we remove it conditionally. I can check with Maria to get her input on this as well.